src/vec.ts

```diff
@@ -15,9 +15,10 @@ export namespace vec {
   export type Vector2 = [x: number, y: number]
 
   /**
-   * A 3x3 matrix (generally used for 2D transformations)
+   * A 2x3 representation of a 3x3 matrix used to transform and translate a
+   * two-dimensional vector.
    */
-  export type Matrix = [number, number, number, number, number, number, number, number, number]
+  export type Matrix = [number, number, number, number, number, number]
```

Comment:
As discussed in Discord, this PR removes the `0 0 1` redundant row from the matrix representation; this change brings the types along for the ride.